packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -298,6 +315,13 @@ class BottomNavigationBar extends StatefulWidget {
     return false;
   }
 
+  static TextStyle _defaultTextStyle(TextStyle textStyle, double fontSize) {
+    textStyle ??= const TextStyle(inherit: false);
+    if (textStyle.fontSize == null)
+      return textStyle.copyWith(fontSize: fontSize);
+    return textStyle;
```

Comment:
Could be:

return textStyle.fontSize == null ? textStyle.copyWith(fontSize: fontSize) : textStyle

Reply:
Done.